docs/fulltext.adoc

```diff
@@ -252,3 +252,29 @@ Again we're search for "Johnny", expecting now two of them:
 CALL apoc.index.search("people", "Johnny") YIELD node, weight
 RETURN node.name, weight
 ----
+
+=== Fulltext Index and multilingual string
+By default standart configuration of fultext index use standart analyzer for extract token from text.
+For support fultext search on multilangual content we need to set additional configuration to parameter `<option>` in function `apoc.index.addAllNodes`.
+
+Make sure `apoc.multilingual.enabled=true` is set.
+
+Bellow there is example of creating fulltext index with multilingual support
+[source,cypher]
+----
+CALL apoc.index.addAllNodes('locations',{
+  Person:  ["address"],
+  Address: ["address"]}, {analyzer:["English","Russian","Thai"]})
```

Comment:
uses analyzer here without an `s`?

Reply:
See my comment for first questions.

Reply:
In this code I found error. After parameter "analizer" I had to write correct class names for analyzer

Reply:
Could you update your PR then with the fixes?